api/package.json

```diff
@@ -20,6 +21,7 @@
     "drizzle-orm": "^0.37.0",
     "express": "^4.21.1",
     "express-session": "^1.18.1",
+    "install": "^0.13.0",
```

Comment:
I assume this is where it's used?

```
nodejs: {
      install: ['@opentelemetry/auto-instrumentations-node'],
    }
```

Reply:
I don't actually think that's necessary and am not sure how that got in - will try removing and seeing if things break

Reply:
yeah it was a copypasta fail. will push a commit removing it. good catch :+1:

![image](https://github.com/user-attachments/assets/7014f8b8-be0a-4d57-842a-c2dab0ec1fc4)